src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1573,14 +1573,14 @@ static LinkAnimationHeader* D_808543D4[] = {
 };
 
 // return type can't be void due to regalloc in func_8084FCAC
-BAD_RETURN(s32) func_80832210(Player* this) {
+BAD_RETURN(s32) Player_ZeroSpeedXZ(Player* this) {
     this->actor.speedXZ = 0.0f;
     this->linearVelocity = 0.0f;
 }
```

Comment:
Worth noting that this conflicts with https://github.com/zeldaret/oot/pull/1346, where this function is named `Player_StopHorizontalMovement`

Reply:
yea have been pondering on that one for a bit. I think a combination of both could work: `_StopHorizontalSpeed`
ZeroSpeedXZ is literally what it does, but saying that its stopping horizontal speed is a bit less technical i guess, and could read nicer when looking at code.
But I think "movement" is going too high level, cause movement can involve a few different factors, not just the literal speed value

Reply:
Fwiw I went with speedXZ mostly because it's shorter

Also this name gets bundled in some other wrapper functions iirc

I'm okay with HorizontalSpeed or HorizSpeed or smth else

Reply:
yea both work. i see ZeroSpeedXZ as the mechanical approach, literally saying what the code does. while horizontal speed is longer, it might help you contextualize what the call means in the places it gets used, rather than having to think "what does setting speedXZ to zero do"

even though its a very obvious operation so it doesnt matter too much, just explaining my thought process

Reply:
So change to `_StopHorizontalSpeed` ?
tbh I'm not a fan of "stop" with "speed", it sounds weird to say "to stop the speed"
and I'd rather have `_StopHorizontalMovement` than `_StopHorizontalSpeed`

my tier list from best to worst (imo, ofc):
- `_ZeroSpeedXZ`
- `_StopHorizontalMovement`
- `_ZeroHorizontalSpeed`
- `_StopMovementXZ`
- `_StopSpeedXZ`
- `_StopHorizontalSpeed`

Also what would this mean for the fantastic anim wrapper names like
`Player_AnimChangeOnceMorphZeroRootYawSpeed` (yes I know it's horrible :( but it basically has to be this, there are a lot of wrappers)
(note "zero" and "speed", with "zero" applying to both "root yaw" and "speed")
`Player_AnimChangeOnceMorphZeroRootYawStopSpeed` ?

Reply:
maybe we unname this for now, since we now need to have the discussion "refer to this as speed or speedXZ"
and naming this at the same time as properly naming `player->linearVelocity` would also be good.

Though I do know some of the anim wrappers do depend on this name, so its a tricky situation lol

Reply:
tbh i am likely going to just open a pr to get this conversation started very soon (today unless I run into issues with naming something)

Reply:
He did it pog
- #1480
so waiting for that to be merged

Reply:
resolving now that 1480 has been merged